wsd/DocumentBroker.cpp

```diff
@@ -1764,11 +1777,10 @@ void DocumentBroker::asyncInstallPresets(const std::shared_ptr<ClientSession>& s
             std::string settings;
             if (FileUtil::Stat(viewSettings).exists())
             {
-                settings =
-                    extractViewSettings(viewSettings, session, _isViewSettingsAccessibilityEnabled);
+                settings = extractViewSettings(viewSettings, session, _isViewSettingsAccessibilityEnabled, _isViewSettingsUpdated);
                 session->sendTextFrame("viewsetting: " + settings);
+                forwardToChild(session, "addconfig");
```

Comment:
This forwardToChild line looks misplaced. It has ended up inside the if (viewSettings ... exists) block when it was originally outside/after it.

Which would suggest that if a user had config data, like autocorr etc files, but no viewsettings file, then the addconfig doesn't get called and they don't get their individualized config activated.